src/components/DownloadPage.tsx

```diff
@@ -42,6 +46,23 @@ export const DownloadPage: React.FunctionComponent<{ teamInfo: TeamInfo }> = ({
                 align-items: center;
             `}
         >
+            {errors.length > 0 && (
+                <Alert
+                    className={css`
+                        margin: 1em;
+                        width: 100%;
+                        max-width: 600px;
+                        text-align: center;
+                    `}
+                    variant="danger"
+                >
+                    <div>There was an issue getting data for the following leagues:</div>
```

Comment:
From Shawn: The error looks good, maybe put "maybe the KHL website is down?" and link to https://isitup.org/krakenhockeyleague.com .